hypersistence-utils-hibernate-60/src/main/java/io/hypersistence/utils/hibernate/id/BatchSequenceGenerator.java

```diff
@@ -141,12 +138,18 @@ public class BatchSequenceGenerator implements BulkInsertionCapableIdentifierGen
 
     /**
      * Indicates the name of the sequence to use, mandatory.
+     * 
+     * @deprecated use {@link BatchSequence}
      */
+    @Deprecated
```

Comment:
We do not really have to deprecate these but since `GenericGenerator` is deprecated in 6.5 and `@BatchSequence` is already available I believe it is justified.